src/Microsoft.Extensions.Configuration.AzureAppConfiguration/JsonConfiguration/JsonConfigurationParser.cs

```diff
@@ -0,0 +1,115 @@
+﻿// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+//
+using Azure.Data.AppConfiguration;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.AzureKeyVault;
+using Microsoft.Extensions.Configuration.AzureAppConfiguration.FeatureManagement;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Mime;
+using System.Text.Json;
+
+namespace Microsoft.Extensions.Configuration.AzureAppConfiguration.JsonConfiguration
+{
+    internal static class JsonConfigurationParser
+    {
+        public static bool IsJsonContentType(string contentType)
+        {
+            if (string.IsNullOrWhiteSpace(contentType))
+            {
+                return false;
+            }
+            string acceptedMainType = "application";
+            string acceptedSubType = "json";
+
+            try
+            {
+                ContentType ct = new ContentType(contentType.Trim().ToLower());
+                var type = ct.MediaType;
+                if (type == FeatureManagementConstants.ContentType ||
+                    type == KeyVaultConstants.ContentType)
```

Comment:
We can extract them in a list as follows, so it is easier to find this list and update it in future if needed.
private static IEnumerable<string> _excludedJsonContentTypes = new [] { FeatureManagementConstants.ContentType, KeyVaultConstants.ContentType };

Reply:
Done

---
In reply to: [454615335](https://github.com/Azure/AppConfiguration-DotnetProvider/pull/181#discussion_r454615335) [](ancestors = 454615335)